### DISCUSSION TITLE
合肥工业大学（宣城校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#37] [University]
> 回答来自群友

**宿舍是上床下桌吗**
上床下桌

**教室和宿舍有没有空调**
有空调，宿舍电费大约8毛一小时

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
没有，公共浴室每层两间，大概2min走到？

**有早自习、晚自习吗**
无早自习，大一有早操、晚自习

**有晨跑吗**
无晨跑，大一有早操

**每学期跑步打卡的要求是多少公里，可以骑车吗**
无要求

**寒暑假放多久，每年小学期有多长**
寒暑假正常时间，今年阴间暑假不足40日

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
可以点外卖，大约5-10MIN路程

**学校交通便利吗，有地铁吗，在市区吗，不在的话进城要多久**
交通出门有公交/出租车，在郊区，进城也没啥可以玩的

**宿舍楼有洗衣机吗**
有，大约0.8-1元一次

**校园网怎么样**
1Gbps的内线，上下行大约500Mbps，按流量收费，0.5元/G，部分节假日免费

**每天断电断网吗，几点开始断**
周日-周四 23点，周五-周六 24点

**食堂价格贵吗，会吃出异物吗**
食堂价格7-8元正常，会有异物，虫子头发都有

**洗澡热水供应时间**
洗澡热水24H供应

**校园内可以骑电瓶车吗，电池在哪能充电**
不可以

**宿舍限电情况**
照明限制大概600W，可以偷空调电（晚上空调不断）

**通宵自习有去处吗**
每个宿舍楼都有一个通宵自习室

**大一能带电脑吗**
可以

**学校里面用什么卡，饭堂怎样消费**
一卡通，全部刷卡

**学校会给学生发银行卡吗**
会

**学校的超市怎么样**
还可以

**学校的收发快递政策怎么样**
有菜鸟裹裹，京东派，邮政在宿舍楼边上，还挺便利的

**学校里面的共享单车数目与种类如何**
哈喽单车，数量不少，但我总是骑不到

**现阶段学校的门禁情况如何**
晚上会有班委查寝，不过大多都是形式

**宿舍晚上查寝吗，封寝吗，晚归能回去吗**
宿舍楼一般12点会关门吧，可以敲门，会被宿舍阿姨骂

**Comment by fautie** [comment]
翡翠湖校区的宿舍楼我只见过一个开放通宵自习室的
不过我也很不提倡,会导致一个卷起来的风气.

**Comment by VRIOUS** [comment]
跑步现在好像又要求了，18级没有，但是我经常看到学弟学妹骑车摇手机😂

**Comment by VRIOUS** [comment]
最后一条还得取决于宿管阿姨，我们阿姨就挺好的😄

**Comment by venti-1** [comment]
大一好像有云运动的跑步要求，算到体育课成绩里面了。

**Comment by mikenihao** [comment]
我是宣城人

**Comment by hyx-621** [comment]
转专业有限制吗
